x-pack/test/functional/services/machine_learning/job_wizard_common.ts

```diff
@@ -295,21 +290,17 @@ export function MachineLearningJobWizardCommonProvider({ getService }: FtrProvid
     },
 
     async ensureAdvancedSectionOpen() {
-      await retry.try(async () => {
+      await retry.tryForTime(5000, async () => {
         if ((await testSubjects.exists('mlJobWizardAdvancedSection')) === false) {
           await testSubjects.click('mlJobWizardToggleAdvancedSection');
-          await testSubjects.existOrFail('mlJobWizardAdvancedSection');
+          await testSubjects.existOrFail('mlJobWizardAdvancedSection', { timeout: 1000 });
         }
       });
     },
 
     async createJobAndWaitForCompletion() {
       await testSubjects.clickWhenNotDisabled('mlJobWizardButtonCreateJob');
-      await retry.waitForWithTimeout(
-        'job processing to finish',
-        5 * 60 * 1000,
-        async () => await testSubjects.exists('mlJobWizardButtonRunInRealTime')
-      );
+      await testSubjects.existOrFail('mlJobWizardButtonRunInRealTime', { timeout: 5 * 60 * 1000 });
```

Comment:
This function is waiting up-to 5 min for job finish. Since we have a mocha timout of 6 min been set for test, in a test with other actions taking ~1 min and job been finished within almost 5 min test will fail without screenshot and html report.
Just FYI you probably want to override default test timeout for such cases. If you know that it is usually loaded faster, I suggest changing 5 min to a smaller number

Reply:
Agreed. ML jobs could need some time to finish, but for the test data we currently have in use a smaller timeout number will work. I will change that in a follow-up.